runtime/kusama/src/lib.rs

```diff
@@ -1076,7 +1064,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
-	(BabeEpochConfigMigrations, KillOffchainPhragmenStorageTest),
+	(KillOffchainPhragmenStorageTest),
```

Comment:
Can you remove this

Reply:
Remove `KillOffchainPhragmenStorageTest`?

Reply:
Yes